spec/features/user_picks_a_move_spec.rb

```diff
@@ -0,0 +1,6 @@
+feature 'Users picks a move' do
+  scenario 'User selects from rock, paper, scissors'do
+  register_and_play
+  click_button 'Rock'
```

Comment:
This test obviously does the job but I think that it's best practice to have an expect clause somewhere in your RSpec tests - in this example could use "expect(page).to have_selector(:link_or_button, 'Rock')" - according to google, I've not tried this matcher yet myself!